### PR TITLE
feat: (e2e tests) Install helm from repo, then upgrade

### LINKED ISSUE #168: add helm upgrade test to ci
**Describe the solution you'd like**
after #108 is merged,
test that installs helm chart using a specific version then upgrades to latest chart is `manifest_staging`

example: https://github.com/open-policy-agent/gatekeeper/blob/master/.github/workflows/upgrade.yaml

### PR DESCRIPTION
**Pull request by pmengelbert**
**What this PR does / why we need it**:
This PR will help ensure that upgrades won't break a cluster.

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Resolves #168

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by pmengelbert** [comment]
It's failing the linter. but passing everything else. This can be rebased once #672 is fixed/merged

File: test/e2e/util/utils.go [hunk 1]
```diff
@@ -213,6 +213,46 @@ func parseRepoTag(img string) (RepoTag, error) {
 	return RepoTag{}, err
 }
 
+func HelmDeployLatestEraserRelease(namespace string, extraArgs ...string) env.Func {
+	return func(ctx context.Context, cfg *envconf.Config) (context.Context, error) {
+		addEraserRepoCmd := exec.Command("bash", "-ec", `
+        helm repo add eraser https://azure.github.io/eraser/charts
```
Comment (sozercan) [inline test/e2e/util/utils.go]: can we make the "https://azure.github.io/eraser/charts" a const or var in case this changes later on?

Comment (pmengelbert) [reply test/e2e/util/utils.go]: done

**Review by sozercan — APPROVED** [review]
Thanks! LGTM

**Pull request merged**